content/spin/v2/testing-apps.md

```diff
@@ -176,21 +276,68 @@ build = "cargo component build --release"
 dir = "tests"
 ```
 
+## Adding Business Logic
+
+The goal of tests is to ensure that the business logic in our application works as intended. For this example, copy and paste the following code into the application's source file ( located at `my-app/my-component/src/lib.rs`):
```

Comment:
```suggestion
The goal of tests is to ensure that the business logic in our application works as intended. We haven't yet updated our "business logic" frrom the "Hello, Fermyon" app. To make our new tests pass, copy and paste the following code into the application's source file (located at `my-app/my-component/src/lib.rs`):
```

(just building out some more context and explanation for why they are doing this)